src/main/frontend/app/components/ReviewInfo.js

```diff
@@ -1,4 +1,4 @@
-import React from "react"
+import React, { useState } from "react"
```

Comment:
Is there a reason you added useState to this component? I do not see it managing state.